crates/apub/src/activities/comment/create_or_update.rs

```diff
@@ -109,14 +109,18 @@ impl ActivityHandler for CreateOrUpdateComment {
   ) -> Result<(), LemmyError> {
     let comment = ApubComment::from_apub(self.object, context, request_counter).await?;
     let recipients = get_notif_recipients(&self.actor, &comment, context, request_counter).await?;
```

Comment:
This is really where the change should be, adding a boolean on sending the email: https://github.com/LemmyNet/lemmy/blob/main/crates/apub/src/activities/comment/mod.rs#L30

Reply:
Ah so its just a problem in the federation code. Honestly the whole websocket/notifications code is so awkward to use, i think i have to do a full rewrite of that.